packages/dashboard-frontend/src/services/workspace-client/devworkspace/__tests__/devWorkspaceClient.create.spec.ts

```diff
@@ -103,6 +104,7 @@ describe('DevWorkspace client, create', () => {
         undefined,
         undefined,
         'eclipse/theia/next',
+        [],
```

Comment:
If the code looks ok, I plan to add more tests in this file

Reply:
The code looks good.